scala/ppml/demo/README.md

```diff
@@ -1,36 +1,63 @@
-# PPML Demo
+# Run PPML Demo in Graphene
```

Comment:
Don't remove current document. That's for non-SGX. Add SGX/Graphene doc below that example.

Reply:
Updated.